.env.testnet.qa

```diff
@@ -1,5 +1,5 @@
 NEXT_PUBLIC_ENV=testnet
-NEXT_PUBLIC_RIF_WALLET_SERVICES=https://frontend.qa.bim.dao.rif.technology/cors_bypass
+NEXT_PUBLIC_RIF_WALLET_SERVICES=https://rws.app.rootstockcollective.xyz
```

Comment:
I think we should use the dev version, because we may be able to manipulate if needed.